andela/vali-ah-frontend
# #167377481 Setup empty react project with webpack and babel

#### What does this PR do?
This pull request sets up an empty react project with webpack and airbnb style guide

#### Description of Task to be completed

 - install and configure webpack and babel
 - install and configure eslint airbnb style guide

#### How should this be manually tested?
- git clone the branch
- cd into the root folder and run 'npm run start:dev'
- your browser should open the landing page 
#### Any Background Context You want to provide?
N/A
#### What are the relevant pivotal tracker stories?

[#167377481](https://pivotaltracker.com/story/show/167377481)

#### Screenshots (if appropriate)

![image](https://user-images.githubusercontent.com/42284402/63702226-73453400-c81e-11e9-81e8-0b48f2c6a486.png)


#### Questions:
